### PR TITLE
Setup v2 triggers for integration tests

### PR DESCRIPTION
**Pull request by taeold**
Also remove region setting option of the test since it's not possible to dynamically set region using function config when parsing triggers via container contract.

Just adding a single v2 trigger def for now - with the structure introduced here, it should be easy to add more triggers in the near future.

### CODE REVIEW

File: integration_test/functions/src/index.ts [hunk 1]
```diff
@@ -107,9 +148,16 @@ function v1Tests(testId: string, accessToken: string) {
...
       .upload('/tmp/' + testId + '.txt'),
-    testLab.startTestRun(firebaseConfig.projectId, testId),
+    testLab.startTestRun(firebaseConfig.projectId, testId, accessToken),
     // Invoke the schedule for our scheduled function to fire
-    callScheduleTrigger('v1-schedule', 'us-central1'),
+    callScheduleTrigger('v1-schedule', 'us-central1', accessToken),
+  ];
+}
+
+function v2Tests(testId: string, accessToken: string) {
```
Comment (inlined) [inline integration_test/functions/src/index.ts:157]: nit: please add return types for functions like this where it may be surprising.

**Review by inlined — APPROVED** [review]
LGTM. We should be able to add GCS tests too by uploading a text file with a testID in it.

File: integration_test/functions/src/index.ts [hunk 2]
```diff
@@ -6,17 +6,18 @@ import * as functions from 'firebase-functions';
 import * as fs from 'fs';
 
 import * as v1 from './v1/index';
-const numTests = Object.keys(v1).length; // Assumption: every exported function is its own test.
-export { v1 };
+import * as v2 from './v2/index';
```
[the remark this answers is not in the document]
Comment (taeold) [reply integration_test/functions/src/index.ts]: Nope!

File: integration_test/functions/src/index.ts (hunk 1 above)
Comment (taeold) [reply integration_test/functions/src/index.ts]: Fixed!

File: integration_test/functions/src/index.ts [hunk 3]
```diff
@@ -6,17 +6,18 @@ import * as functions from 'firebase-functions';
...
+const numTests = Object.keys(v1).length + Object.keys(v2).length; // Assumption: every exported function is its own test.
+export { v1, v2 };
 
 import * as testLab from './v1/testLab-utils';
+import { REGION } from './region';
 
 const firebaseConfig = JSON.parse(process.env.FIREBASE_CONFIG);
-const REGION = functions.config().functions.test_region;
```
[the remark this answers is not in the document]
Comment (taeold) [reply integration_test/functions/src/index.ts:17]: But v2 deploys relies on container contract, and it does not inject runtime config during trigger parsing, this dynamic region is doomed. I think we should revive ability to test in different regions once params is a thing.

**Comment by taeold** [comment]
@inlined Yep. I'll do that in following PRs (when the time comes!)

**Pull request merged**